### PR TITLE
[Translation] added missing Albanian translations

### LINKED ISSUE #38722: Missing translations for Albanian (sq)
Hello,

There are some missing translation for Albanian (sq). These should be added in branch 3.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Form/Resources/translations/validators.sq.xlf](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Form/Resources/translations/validators.sq.xlf)
- [src/Symfony/Component/Security/Core/Resources/translations/security.sq.xlf](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Security/Core/Resources/translations/security.sq.xlf)
- [src/Symfony/Component/Validator/Resources/translations/validators.sq.xlf](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Validator/Resources/translations/validators.sq.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by abame**
[Translation] added missing Albanian translations for the following components:

1. [Forms]
2. [Security][Core]
3. [Validator]

| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #38722
| License       | MIT

### CODE REVIEW

File: src/Symfony/Component/Form/Resources/translations/validators.sq.xlf [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+                <source>Please enter an integer.</source>
+                <target>Ju lutemi shkruani një numër të plotë.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Please select a valid language.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Ju lutemi zgjidhni një lokale të vlefshme.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.sq.xlf:75]: Can you double check if this is correct?

File: src/Symfony/Component/Security/Core/Resources/translations/security.sq.xlf [hunk 2]
```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="1">
+                <source>An authentication exception occurred.</source>
+                <target>Ndodhi një problem në verifikim.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Security/Core/Resources/translations/security.sq.xlf]: Can you check this one too?

File: src/Symfony/Component/Security/Core/Resources/translations/security.sq.xlf [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            </trans-unit>
+            <trans-unit id="2">
+                <source>Authentication credentials could not be found.</source>
+                <target>Kredencialet e verifikimit nuk mund të gjendeshin.</target>
+            </trans-unit>
+            <trans-unit id="3">
+                <source>Authentication request could not be processed due to a system problem.</source>
+                <target>Kërkesa për verifikim nuk mund të përpunohej për shkak të një problemi në sistem.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Security/Core/Resources/translations/security.sq.xlf]: Is this "authentication" or "verification"?

Same with number 2.

File: src/Symfony/Component/Security/Core/Resources/translations/security.sq.xlf [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+                <source>Account is locked.</source>
+                <target>Llogaria është e kyçur.</target>
+            </trans-unit>
+            <trans-unit id="17">
+                <source>Too many failed login attempts, please try again later.</source>
+                <target>Shumë përpjekje të dështuara autentikimi, provo përsëri më vonë.</target>
+            </trans-unit>
+            <trans-unit id="18">
+                <source>Invalid or expired login link.</source>
+                <target>Link i pavlefshëm ose i skaduar.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Security/Core/Resources/translations/security.sq.xlf]: Does it say "login link"?

**Review by Nyholm — COMMENTED** [review]
Cool. Thank you

I used Google translate to try to verify these. My comments below might be wrong, but could you check?

File: src/Symfony/Component/Form/Resources/translations/validators.sq.xlf (hunk 1 above)
Comment (abame) [reply src/Symfony/Component/Form/Resources/translations/validators.sq.xlf:75]: Yes, this is correct. Just as an explanation, from Google Translate the word `lokal` means a place where you can have a coffee and the correct word is `lokale` in this context

File: src/Symfony/Component/Security/Core/Resources/translations/security.sq.xlf (hunk 2 above)
Comment (abame) [reply src/Symfony/Component/Security/Core/Resources/translations/security.sq.xlf]: authentication and verification in Albanian are kind of similar but will update it to match exactly to authentication

**Comment by abame** [comment]
I don't understand why the typo check is failing when that's the correct word in Albanian.

File: src/Symfony/Component/Form/Resources/translations/validators.sq.xlf (hunk 1 above)
Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.sq.xlf:75]: Okey. Cool!

**Comment by Nyholm** [comment]
It is failing because Fabbot only speaks English. We can ignore that. =)

File: src/Symfony/Component/Security/Core/Resources/translations/security.sq.xlf [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="1">
+                <source>An authentication exception occurred.</source>
+                <target>Ndodhi një problem në autentikim.</target>
+            </trans-unit>
+            <trans-unit id="2">
+                <source>Authentication credentials could not be found.</source>
+                <target>Kredencialet e verifikimit nuk mund të gjendeshin.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Security/Core/Resources/translations/security.sq.xlf]: Maybe use "autentikim" here too?

**Review by Nyholm — APPROVED** [review]
Excellent job. 

Im super happy with this now
👍

**Comment by nicolas-grekas** [comment]
Thank you @abame.

**Pull request merged**